src/pages/PageAbout.jsx

```
@@ -2,15 +2,16 @@ import React from "react";
-      <h1>About MovieFix</h1>
-      <p>Welcome to MovieFix, your ultimate destination for all things movies! whether you're searching for the latest blockbusters, timeless classics, or hidden gems, MovieFix has you covered. Explore detailed reviews, cast and crew insights, trailers, and personalized recommendations tailored to your taste.</p>
-      <p>
-        Our Mission is to help you discover your next favorite film, one fix at a time.
-      </p>
-      <p>Dive in, and let your cinematic journey begin!</p>
-    </section>
+    <main className="text-(--color-neutral-light) flex flex-col items-center min-h-screen px-1">
+      <section className="text-center max-w-3xl mt-50 sm:mt-50">
+        <h1 className="text-2xl md:text-4xl font-bold mb-5">About MovieFix</h1>
+        <p className="text-base md:text-lg leading-relaxed mb-3 max-w-[90%] sm:max-w-[700px] mx-auto">
+          Welcome to MovieFix, your ultimate destination for all things movies! Whether you're searching for the latest blockbusters, timeless classics, or hidden gems, MovieFix has you covered. Explore detailed reviews, cast and crew insights, trailers, and personalized recommendations tailored to your taste.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)